pilot/pkg/bootstrap/configcontroller.go

```diff
@@ -142,7 +142,14 @@ func (s *Server) initConfigController(args *PilotArgs) error {
 	s.environment.IstioConfigStore = model.MakeIstioStore(s.configController)
 
 	if features.EnableIOR {
-		ior.Register(s.kubeClient, s.configController, args.Namespace, s.mrc)
+		s.addStartFunc(func(stop <-chan struct{}) error {
+			go leaderelection.
+				NewLeaderElection(args.Namespace, args.PodName, leaderelection.StatusController, s.kubeClient).
```

Comment:
If an Istiod instance is running IOR and another gets elected, does this need to stop?

Reply:
nope, I have to implement that. Thanks for the catch.

Reply:
done